examples/cotire/CMakeLists.txt

```diff
@@ -0,0 +1,17 @@
+# Copyright (c) 2017, Pawel Bylica
+# All rights reserved.
+
+cmake_minimum_required(VERSION 3.0)
+
+# Emulate HunterGate:
+# * https://github.com/hunter-packages/gate
+include("../common.cmake")
+
+project(download-cotire)
+
+# download cotire
+hunter_add_package(cotire)
+message(${COTIRE_ROOT})
+
+# This works, but is quite ugly. Can we get include(cotire) to work?
```

Comment:
You can create `cotireConfig.cmake`. Take a look at Gauze:
* https://github.com/hunter-packages/gauze/blob/master/cmake/Config.cmake.in